package.json

```diff
@@ -18,7 +18,7 @@
     "clean": "rimraf lib lib-esm",
     "eslint": "eslint src --cache",
     "eslint-fix": "npm run eslint -- --fix",
-    "playwright": "playwright test -c playwright.config.ts",
+    "playwright": "playwright test -c playwright.config.ts --update-snapshots",
```

Comment:
We should not update snapshots during the tests ! They should be store in the project but we need to generate the snapshot for linux.

Reply:
fixed thanks